app/src/app.js

```diff
@@ -0,0 +1,47 @@
+const postModel = require('./model/post');
+const express = require('express');
+const connection = require('./model/connection');
+const app = express();
+const bodyParser = require('body-parser');
+app.use(bodyParser.json());
+
+app.post('/posts', async (req, res) => {
+
+    let addPost = new postModel.PostEntity(null, req.body.title, req.body.slug, req.body.content);
```

Comment:
Teraz jeszcze mi to wpadło w oko. Nazwałaś rzecz czasownikiem. Jak coś to powinnaś to nazwać `addedPost`, chociaż imho `newPost` brzmi naturalniej

Reply:
a mogę to dać jako nowy commit? :D

Reply:
jasne, nie inaczej

Reply:
poprawione